src/resource.coffee

```diff
@@ -91,14 +91,14 @@ class Resource extends Model
     else
       throw new Error 'Can\'t uncache a resource with no ID'
 
-  delete: ->
+  delete: (query={})->
```

Comment:
Any idea why this missing param hasn't failed before? Adding the default arg here makes sense but it's concerning that it's never raised an error before...or has it?

Reply:
I'm baffled.

Reply:
i guess we fix it and move on with our work.